caasp-velero-image/config.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#======================================
+# Functions...
+#--------------------------------------
+test -f /.kconfig && . /.kconfig
+test -f /.profile && . /.profile
+
+#======================================
+# Greeting...
+#--------------------------------------
+echo "Configure image: [$kiwi_iname]..."
+
+#======================================
+# Imitate upstream image:
+# velero/velero
+# same file structure
+#--------------------------------------
+cp /usr/bin/velero /velero
```

Comment:
Why not a symlink? this adds about 50Mb of extra data into the image. Go packages are, in many cases, horribly fat.

Reply:
Good idea. Updated!